### PR TITLE
fix: rosetta construction derive api

### PR DESCRIPTION
**Pull request by rllola**
## Description

Update the rosetta `construction/derive` api to use `account_identifier` and not the deprecated `address`. See rosetta spec (https://www.rosetta-api.org/docs/models/ConstructionDeriveResponse.html)

It also update npm package axios because of a `Server-Side Request Forgery` vulnerability in the current version of the package.
 
For details refer to issue #10 

The modifications include:
- [x] update npm package axios
- [x] updating the schema file for the rosetta construction derive file
- [x] generating new typescript files
- [x] updating the api implementation to return the proper response
- [x] updating the test for this specific endpoint

### CODE REVIEW

File: client/src/generated/apis/AccountsApi.ts [hunk 1]
```diff
@@ -162,7 +162,7 @@ export class AccountsApi extends runtime.BaseAPI implements AccountsApiInterface
             throw new runtime.RequiredError('principal','Required parameter requestParameters.principal was null or undefined when calling getAccountAssets.');
         }
 
-        const queryParameters: runtime.HTTPQuery = {};
+        const queryParameters: any = {};
```
Comment (rllola) [inline client/src/generated/apis/AccountsApi.ts]: I don't understand why it is generating me this new type here when I haven't done any changes in those files.

**Comment by jleni** [comment]
Do we want to commit all generated files? What was the feedback about this?

**Comment by rllola** [comment]
> Do we want to commit all generated files? What was the feedback about this?

The feedback that I got is that those files are added by the CI when it is running. According to them there is room for improvement so maybe something we can figure out with them. For now I would just follow their way of doing and remove them.

**Comment by rllola** [comment]
I have removed them and it is cleaner now. The unexpected modifications in the generated files are gone.

**Comment by jleni** [comment]
Nice!

**Pull request merged**